api/app/Models/User.php

```diff
@@ -2,38 +2,62 @@
 
 namespace App\Models;
 
+use BeatSwitch\Lock\LockAware;
+use BeatSwitch\Lock\Callers\Caller;
 use Illuminate\Auth\Authenticatable;
 use Illuminate\Contracts\Auth\Authenticatable as AuthenticatableContract;
 
-class User extends BaseModel implements AuthenticatableContract
+class User extends BaseModel implements AuthenticatableContract, Caller
 {
-    use Authenticatable;
+    use Authenticatable, LockAware;
 
     const USER_TYPE_ADMIN = 'admin';
     const USER_TYPE_PUBLIC = 'public';
```

Comment:
I realise I defined this value, but after looking at the naming convention from Lock, I think it makes more sense to have the `public` user type be `guest`.